sign-in.txt

```diff
@@ -10,9 +9,11 @@ TODO: Use the terminal and git commands to complete the PULL; MERGE; COMMIT; PUS
 
   NAME  : GitHub Username
   -----------------------------
-  Brian  : Bansenauer-Cascadia
-  Shahin : shahinnezam
-  Andrea : Andrealharrison
```

Comment:
Fabrice, it looks like you confused your base code with the base code pulled from the develop branch. The develop branch's code is our team baseline, so when you pull and merge code from develop, try to keep all the incoming code, and make only the changes needed to add your feature item (in this case your name) into the code.